pe-parser-library/src/parse.cpp

```diff
@@ -2655,20 +2655,35 @@ bool GetDataDirectoryEntry(parsed_pe *pe,
     return false;
   }
 
-  section sec;
-  if (!getSecForVA(pe->internal->secs, addr, sec)) {
-    PE_ERR(PEERR_SECTVA);
-    return false;
-  }
+  /* NOTE(ww): DIR_SECURITY is an annoying special case: its contents
+   * are never mapped into memory, so its "RVA" is actually a direct
+   * file offset.
+   */
```

Comment:
I think it would be useful to paste a URL with a source for this comment

Reply:
I think it should be this link, right? https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#the-attribute-certificate-table-image-only

Reply:
Yep, that looks right. Will add.